### PR TITLE
chore: support script to run on Windows

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description

Previously, developer who use Windows for development cannot run any script in the project. The root cause is incorrect of protocol of URL.

## Type of change

- [x] Refactor (improves code without changing its functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: scripts/release/versioning.mjs [hunk 1]
```diff
@@ -16,7 +16,7 @@ const PLACEHOLDER_VERSION = 'PUBLISH_VERSION';
 const FILE_OPTIONS = { encoding: 'utf8' };
 
 const handler = async () => {
-  const packageJson = await getJSON(path.resolve(PACKAGE_ROOT, 'package.json'), import.meta);
```
Comment (Nantawat-Poothong) [inline scripts/release/versioning.mjs]: What is the `import.meta`? seems like you didn't need it anymore.

**Comment by Nantawat-Poothong** [comment]
Did you test on linux side as well? Is that smooth?

**Comment by bualoy-napat** [comment]
> Did you test on linux side as well? Is that smooth?

Yes, I did. it looks good on Linux.

File: scripts/helpers/esm.mjs [hunk 2]
```diff
@@ -16,13 +16,14 @@ export const fileDirName = meta => {
...
- * @param string meta the base URL to resolve against if the input is not absolute
+ * @param url the absolute or relative input URL to parse
+ * @param meta the base URL to resolve against if the input is not absolute
  * @returns object
  */
 export const getJSON = async (url, meta = undefined) => {
+  const _url = pathToFileURL(url);
   return JSON.parse(
-    await fs.promises.readFile(new URL(url, meta ? meta.url : undefined))
+    await fs.promises.readFile(new URL(_url, meta ? meta.url : undefined))
```
Comment (Nantawat-Poothong) [inline scripts/helpers/esm.mjs:26]: Is it necessary to define the `_url` variable?

**Review by Nantawat-Poothong — APPROVED** [review]
LGTM

**Pull request merged**